ambry-store/src/main/java/com.github.ambry.store/StoreMessageReadSet.java

```diff
@@ -111,14 +120,14 @@ public int compareTo(BlobReadOptions o) {
 
   byte[] toBytes() {
```

Comment:
please add a comment pointing out the deficiencies of this function (what it doesn't serialize)